### PR TITLE
Improved MicroProfile property value expression diagnostic message

### PR DESCRIPTION
**Pull request by AlexXuChen**
Improved MicroProfile property value expression diagnostic message.

Signed-off-by: Alexander Chen <alchen@redhat.com>

### CODE REVIEW

File: microprofile.ls/org.eclipse.lsp4mp.ls/src/main/java/org/eclipse/lsp4mp/services/properties/PropertiesFileValidator.java [hunk 1]
```diff
@@ -223,35 +223,35 @@ private void validatePropertyValueExpressions(String propertyName, ItemMetadata
...
 							} else {
 								// The expression has default value (ex : ${DBUSER:sa}) otherwise the error
 								// is reported
 								Range range = PositionUtils.createRange(propValExpr.getReferenceStartOffset(),
-										propValExpr.getReferenceEndOffset(), propValExpr.getDocument());
+									propValExpr.getReferenceEndOffset(), propValExpr.getDocument());
 								if (range != null) {
-									addDiagnostic("Unknown referenced property '" + refdProp + "'", range,
-											expressionSeverity, ValidationType.expression.name());
+									addDiagnostic("Unknown referenced property value expression '" + refdProp + "'",
```
[the remark this answers is not in the document]
Comment (angelozerr) [reply microprofile.ls/org.eclipse.lsp4mp.ls/src/main/java/org/eclipse/lsp4mp/services/properties/PropertiesFileValidator.java:253]: OK please update the title of the pr  with "improve diagnostic message"

[the remark this answers is not in the document]
Comment (AlexXuChen) [reply microprofile.ls/org.eclipse.lsp4mp.ls/src/main/java/org/eclipse/lsp4mp/services/properties/PropertiesFileValidator.java:253]: Updated

**Comment by fbricon** [comment]
message lgtm

**Comment by angelozerr** [comment]
Thanks @AlexXuChen

**Pull request merged**